Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat

```diff
@@ -133,7 +134,7 @@ Material:
     - _ZWrite: 0
     m_Colors:
     - _BaseColor: {r: 0.6985294, g: 0.82535493, b: 1, a: 0.24705882}
-    - _Color: {r: 1, g: 1, b: 1, a: 1}
+    - _Color: {r: 0.6985294, g: 0.82535493, b: 1, a: 0.24705882}
```

Comment:
Why did the colours need to change?

Reply:
Uhhhhh I didn't even know I touched this

Reply:
Then please revert it. If you need to, grab a copy from `develop`.